src/geovista/examples/warp/from_unstructured__fvcom.py

```diff
@@ -13,7 +13,7 @@
 📋 Summary
 ^^^^^^^^^^
 
-Creates a mesh from 1-D latitude and longitude unstructured cell points.
+Creates a mesh from 1-D latitude and longitude unstructured data.
```

Comment:
```suggestion
Creates a mesh from 1-D latitude and longitude unstructured points.
```

Reply:
My problem with this is, they aren't "just" isolated points, because the source data does tell you how they connect into cells, which is essential info for constructing the mesh ...

Reply:
Could we use something like "from the 1-D unstructured  latitude and longitude points and connectivity" ?

Reply:
I guess if we were just using the points and maybe triangulating them, the existing statement would work for me.  But we're not actually doing that.

Reply:
No triangulation is involved here.

The `connectivity` (shape `(N, 3)`) is used to index into the points (1-D `lons` and `lats` both with shape `(M,)`) in order to create the faces of the mesh, which all happen to be triangular.

That's what's happening. Do you think something else is involved here?

Reply:
> No triangulation is involved here ... Do you think something else is involved?

No, that's actually my point :   There is a given face connectivity in the source data, and we use that to build our mesh :  So it is  **_not_** based purely on the point locations, there is additional info present in the source data.

Reply:
I've already agreed with you to say `points and connectivity` ...